asv_bench/benchmarks/io/style.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+
+from pandas import DataFrame
+
+
+class RenderApply:
+
+    params = [[12, 24, 36], [12, 120, 1200]]
```

Comment:
you likely don't need to test the 1200 one as the other benchmarks show enough. we want a benchmark to take < 1s

Reply:
I see your point.

I can remove 1200 case but will you suggest how I can tell this:
Original reason I looked into this issue is because my app was suffering a delay more than 20 seconds. It used MultiIndex rows and columns so it was more complicated but it went from 20 seconds down to 2 seconds.
If you see the #19917 the complaint is that the styler is "extremely" slow. So the applications that are suffering probably are all with big size tables.

I'd love to have this case removed because running the benchmark test really took long time on me - it never ended on my computer until hours later when I figured out how I run a single benchmark.

But at the same time wouldn't it be necessary to put the comparison somewhere may be in form of comments?

BTW, this benchmark took about 7 minutes to compare results on two different revisions.

Reply:
I left a comment in #19917 - this should be enough I guess.

Reply:
right the key is how long do these benchmarks take after your patch?